b2b-sample/libs/business-admin-app/data-access/data-access-controller/src/lib/controller/identityProvider/controllerCreateIdentityProvider/controllerDecodeCreateIdentityProvider.ts

```diff
@@ -31,11 +47,33 @@ import { controllerCallCreateIdentityProvider } from "./controllerCallCreateIden
  * @returns - whether the identity provider is created or not
  */
 export async function controllerDecodeCreateIdentityProvider(session: Session, template: IdentityProviderTemplate,
-    formValues: Record<string, string>): Promise<IdentityProvider | null> {
+    formValues: Record<string, string>, configureType?: IdentityProviderConfigureType)
+    : Promise<IdentityProvider | null> {
 
     let model: IdentityProviderTemplateModel = JSON.parse(JSON.stringify(template.idp));
 
-    model = setIdpTemplate(model, template.templateId, formValues, (session.orgId as string));
+    if (configureType) {
+        switch (configureType) {
+            case IdentityProviderConfigureType.AUTO: {
+                const identityProviderDiscoveryUrl: IdentityProviderDiscoveryUrl | null
+                    = await getIdentityProviderFromDiscoveryUrl(session, formValues["discovery_url"].toString());
+
+                if (identityProviderDiscoveryUrl) {
+                    model = setIdpTemplate(model, (template.templateId as string), formValues, (session.orgId as string)
+                        , identityProviderDiscoveryUrl);
+                }
+
+                break;
+            }
+
+            case IdentityProviderConfigureType.MANUAL: {
+
+                model = setIdpTemplate(model, (template.templateId as string), formValues, (session.orgId as string));
+
```

Comment:
Let's remove unwanted new lines. Check other places too.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/c9fb90d7d7ab19441f2e0d135cf2a6f218b7c78b